### DISCUSSION TITLE
garbo purchases

### DISCUSSION THREAD

**Discussion by chocofungi** [loathers/garbage-collector#1980] [Q&A]
is there a command i can use to see what items garbo is purchasing? it's not included in the round up at the end of the script running and i'd like to know what it's buying. thanks in advance for any info

**Comment by gausie** [comment]
I don’t understand, the gCLI tells you what garbo is purchasing. How could you run a command?

**Reply by gausie** [reply]
Oh I see, do you mean the present habitual i.e. “items garbo is in the habit of purchasing”?

That does not exist and we have no plans to add it.

**Reply by chocofungi** [reply]
to know what its purchasing whilst it does it i would have to sit and watch the script run which would be very boring and take a lot of time. i am asking can i find out at the end of the session what things it purchased during the session or is that information lost?

**Reply by spaghetti-squash** [reply]
You can check kolmafia session logs for strings of the form `buy 1 milk of magnesium for 1944 each from shop #294980 on 20240807`
